R/staggregate.R

```diff
@@ -260,6 +299,16 @@ polygon_aggregation <- function(clim_dt, weights_dt, list_names, time_agg){
 #' @param time_agg the temporal scale to aggregate data to. Options are
 #'   `'hour'`, `'day'`, `'month'`, or `'year'` (`'hour'` cannot be selected
 #'   unless `daily_agg = 'none'`)
+#' @param start_date the date (and time, if applicable) of the first layer in
+#'  the raster. To be input in a format compatible with
+#'  lubridate::as_datetime(), e.g. `"1991-10-29"` or `"1991-10-29 00:00:00"`.
+#'  The default is `NA` since the rasters usually already contain temporal
+#'  information in the layer names and they do not need to be manually supplied.
+#' @param time_interval the time interval between layers in the raster to be
+#'  aggregated. To be input in a format compatible with seq(), e.g.
+#'  `'1 day'` or `'3 months'`. The default is `'1 hour'` and this argument is
+#'  required if daily_agg is not `'none'` or if the layer name format is not
+#'  compatible with stagg.
```

Comment:
Maybe we can update just this last part of "or if layer name format is not compatible with stagg" to be "or if the start_date argument is not NA" (as long as that is true)